prime-router/src/main/java/fhirengine/translation/hl7/structures/fhirinventory/group/OML_O21_OBSERVATION_REQUEST.java

```diff
@@ -35,11 +35,10 @@
 import ca.uhn.hl7v2.HL7Exception;
 import ca.uhn.hl7v2.model.AbstractGroup;
 import ca.uhn.hl7v2.model.Group;
-import ca.uhn.hl7v2.model.v27.group.OML_O21_PRIOR_RESULT;
 import ca.uhn.hl7v2.model.v27.segment.CTD;
-import ca.uhn.hl7v2.model.v27.segment.DG1;
 import ca.uhn.hl7v2.model.v27.segment.TCD;
 import ca.uhn.hl7v2.parser.ModelClassFactory;
+import fhirengine.translation.hl7.structures.fhirinventory.segment.DG1;
```

Comment:
The decisions to create a new class are based entirely on whether the HAPI 2.7 classes are fully capable of storing all fields specified in 2.5.1. To take RXO as an example, we did not create a version of the segment because the 2.7 class can fully represent a 2.5.1 message (CE is represented by CWE, and there are no NULLDT segments). The segments that we created classes for are in the cases where the HAPI 2.7 classes were deficient in some way; this can only be determined on a case by case basis.

Reply:
Thanks for the explanation!